tap_google_sheets/transform.py

```diff
@@ -145,37 +145,77 @@ def transform_sheet_boolean_data(value, unformatted_value, sheet_title, col_name
         return col_val
 
 # transform decimal values in the sheet
-def transform_sheet_decimal_data(value, sheet_title, col_name, col_letter, row_num, col_type):
+def transform_sheet_decimal_data(value_type, formatted_value, unformatted_value, sheet_title, col_name, col_letter, row_num, col_type):
+    """
+        Transform number type data and return according to the datatype in the sheet
+
+        :param
+        value_type: the datatype of unformatted value ie. int or float
+        formatted_value - The displayed value of a cell in the sheet
+        unformatted_value - The formatted value of a cell in the sheet
```

Comment:
@harshpatel4crest Is the description of unformatted value correct? Can you mention which value will be original and which value will be abstract in the sheet?

Reply:
Added an example for unformatted and formatted values from Google's API.